packages/editor/src/components/post-actions/actions.js

```diff
@@ -491,6 +499,273 @@ export const renamePostAction = {
 	},
 };
 
+export const resetTemplateAction = {
```

Comment:
(I don't remember if I have already asked this question) Why are we exporting these actions? Unless there is a present need, let's hide them and only expose `usePostActions`, no?

Reply:
Yes, I missed this on the other PR, the exports were removed.